cluster-controller/src/main/java/io/strimzi/controller/cluster/resources/Labels.java

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.controller.cluster.resources;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.singletonMap;
+import static java.util.Collections.unmodifiableMap;
+
+/**
+ * An immutable set of labels
+ */
+public class Labels {
+
+    public static final String STRIMZI_DOMAIN = "strimzi.io/";
+
+    /**
+     * The kind of a ConfigMap:
+     * <ul>
+     *     <li>{@code strimzi.io/kind=cluster}
+     *         identifies a ConfigMap that is intended to be consumed by
+     *         the cluster controller.</li>
+     *     <li>{@code strimzi.io/kind=topic}
+     *         identifies a ConfigMap that is intended to be consumed
+     *         by the topic controller.</li>
+     * </ul>
+     */
+    public static final String STRIMZI_KIND_LABEL = STRIMZI_DOMAIN + "kind";
+    /**
+     * The type of Strimzi component:
+     * E.g: {@code kafka}, {@code zookeeper}, {@code topic-controller},
+     *      {@code connect}, {@code connect-s2i}
```

Comment:
The type we use in ConfigMaps is `kafka-connect` and `kafka-connect-s2i`. Is this a typo? Or are we using different names for marking the resources we create?